Answers.md

```diff
@@ -0,0 +1,63 @@
+**What is React JS? How does it differ from other JavaScript Frameworks?**
+
+*First of all, React is a Library, not a Framework, for very specific reasons. The React library itself doesn't look to solve any structural or architechtural problems in an application -- it's completely structure agnostic. All it does is use a component-based idealogy to help break user interfaces down into reusable components that can be used all over websites. React doesn't care how you implement it or what you do with that power -- it's just there to help you do what you need to do. Frameworks tend to define entire application ecosystems.*
```

Comment:
You are very correct. Notice I didn't ask to describe the ReactJS framework, but why its different from other JS Frameworks. But you answered this very well!